src/components/UpcomingEvents/index.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import moment from 'moment'
+import { FormattedMessage } from 'react-intl'
+import { Row, Col } from 'reactstrap'
+import './index.css'
+import events from './events'
+
+const Milestone = ({ event }) => {
+  const style = {
+    backgroundColor: event.backgroundColor,
+    minWidth: '272px',
+    height: '127px',
+    marginLeft: '10px',
+  }
+
+  return (
+    <div
+      className="card--content event p-2"
+      style={{ ...style }}
+    >
+      <h3 className="event-header" style={{ color: event.headerColor }}>
+        <FormattedMessage id={event.title} />
+      </h3>
+      <ul className="events-container">
+        {event.descriptions.map(desc => (
+          <li className="event" key={desc}>
+            <FormattedMessage id={desc} />
+          </li>
+        ))}
+      </ul>
+    </div>
+  )
+}
+
+export default class UpcomingEvents extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      value: 0,
```

Comment:
state is not used at all. can be removed, then this component can be refactored to a stateless functional component